lib/hammer_cli/apipie/command.rb

```diff
@@ -38,7 +38,7 @@ def execute
 
     def help
       help_str = super
-      if !resource || (!action.nil? && !resource.has_action?(action))
+      if !self.class.resource || (!action.nil? && !self.class.resource.has_action?(action))
```

Comment:
I think the original version is correct. Instance method `resource` is defined here https://github.com/theforeman/hammer-cli/blob/master/lib/hammer_cli/apipie/resource.rb#L9 and is properly called. The problem seems to be in https://github.com/theforeman/hammer-cli/blob/master/lib/hammer_cli/apipie/resource.rb#L10 where the `self.parent_command.class.resource` is not defined (MainCommand is not successor of ApipieCommand and has no resource). The proper fix should likely be to check for existence of the resource method there.

I've update the issue with explanation of what is causing the error.